test/unit/registration.js

```diff
@@ -51,34 +53,24 @@ describe('Returning a constructor', function () {
   });
 
   it('should affect the element prototype even if it was not constructed using the constructor.', function () {
-    var Div = skate('div', {
+    var tag = helpers.safeTagName('my-el');
+    var Element = skate(tag.safe, {
       prototype: {
         func1: function () {}
       }
     });
 
-    Div.prototype.func2 = function () {};
-
-    var div = new Div();
-
-    expect(div.func1).to.be.a('function');
-    expect(div.func2).to.be.a('function');
-  });
-
-  it('should allow the overwriting of the prototype', function () {
```

Comment:
We could write a test to make sure the prototype cannot be overwritten, but I'm not sure if we need to go that far. It's just not a feature anymore.